client/dashboard/store-performance/index.js

```diff
@@ -77,33 +77,46 @@ class StorePerformance extends Component {
 				<SectionHeader title={ __( 'Store Performance', 'wc-admin' ) } menu={ this.renderMenu() } />
 				<Card className="woocommerce-dashboard__store-performance">
 					<SummaryList>
-						{ showCustomers && (
-							<SummaryNumber
-								label={ __( 'New Customers', 'wc-admin' ) }
-								value={ '2' }
-								prevLabel={ __( 'Previous Week:', 'wc-admin' ) }
-								prevValue={ 3 }
-								delta={ -33 }
-							/>
-						) }
-						{ showProducts && (
-							<SummaryNumber
-								label={ __( 'Total Products', 'wc-admin' ) }
-								value={ totalProducts }
-								prevLabel={ __( 'Previous Week:', 'wc-admin' ) }
-								prevValue={ totalProducts }
-								delta={ 0 }
-							/>
-						) }
-						{ showOrders && (
-							<SummaryNumber
-								label={ __( 'Total Orders', 'wc-admin' ) }
-								value={ totalOrders }
-								prevLabel={ __( 'Previous Week:', 'wc-admin' ) }
-								prevValue={ totalOrders }
-								delta={ 0 }
-							/>
-						) }
+						{ () => {
+							const summaryNumbers = [];
+							if ( showCustomers ) {
+								summaryNumbers.push(
+									<SummaryNumber
+										key="1"
+										label={ __( 'New Customers', 'wc-admin' ) }
+										value={ '2' }
+										prevLabel={ __( 'Previous Week:', 'wc-admin' ) }
+										prevValue={ 3 }
+										delta={ -33 }
+									/>
+								);
+							}
+							if ( showProducts ) {
+								summaryNumbers.push(
+									<SummaryNumber
+										key="2"
+										label={ __( 'Total Products', 'wc-admin' ) }
+										value={ totalProducts }
+										prevLabel={ __( 'Previous Week:', 'wc-admin' ) }
+										prevValue={ totalProducts }
+										delta={ 0 }
+									/>
+								);
+							}
+							if ( showOrders ) {
+								summaryNumbers.push(
+									<SummaryNumber
+										key="3"
+										label={ __( 'Total Orders', 'wc-admin' ) }
+										value={ totalOrders }
+										prevLabel={ __( 'Previous Week:', 'wc-admin' ) }
+										prevValue={ totalOrders }
+										delta={ 0 }
+									/>
+								);
+							}
+							return summaryNumbers;
+						} }
```

Comment:
This is ugly, but its getting redone in https://github.com/woocommerce/wc-admin/pull/1343 anyways